packages/kotti-ui/source/kotti-style/_loadings.scss

```diff
@@ -14,41 +14,42 @@
 	}
 }
 
-@mixin skeleton-style($bg-x, $bg-y) {
+%skeleton-style {
 	box-sizing: border-box;
 	display: block;
 	width: 100%;
-	margin: 0.4rem 0;
```

Comment:
intentional

Reply:
Design team agreed to remove this and instead make sure that two direct loadings have a separation margin of 0.4rem and not 0.8rem (which maintains the previous case)